src/test/name_applications_tests.cpp

```diff
@@ -128,4 +128,14 @@ BOOST_AUTO_TEST_CASE( valid_json )
     BOOST_CHECK_EQUAL(IsValidJSONOrEmptyString(""), true);
 }
 
+BOOST_AUTO_TEST_CASE( minimal_json){
+    BOOST_CHECK_EQUAL(IsMinimalJSONOrEmptyString("{\"bar\":[1,2,3]}"), true);
+
+    BOOST_CHECK_EQUAL(IsMinimalJSONOrEmptyString("{\"bar\":  [1,2,3]}"), false);
```

Comment:
Another case you could check for is `{\"bar\":[1, 2, 3]}`, i.e. whitespace added around the commas.  This is not something that could specifically happen with Univalue's `write` as I understand it, but for instance the way Python serialisation is configured, a wrong config could produce this.  It can't hurt to have it in the test, too, just in case.

Reply:
Looks like Rose has added this; look OK to you Daniel?